config.go

```diff
@@ -45,6 +45,8 @@ type Config struct {
 	LogOutput io.Writer
 	// HTTPClient is the http client
 	HTTPClient *http.Client
+	// Specify an Explicit Marathon Version
+	Version string
```

Comment:
`marathon.Config` is meant to hold client-side options. `Version` is only relevant for the (fake) server end, however. It could mislead client users into thinking that they can make use the field somehow.

My suggestion would be to change [newFakeMarathonEndpoint](https://github.com/kevinschoon/go-marathon/blob/2bdb699c4c6b14c5b58a92879c9fc626d5dec157/testing_utils_test.go#L84)'s signature such that it accepts a (newly defined) configuration container struct holding both client-side and server-side configurations. The former would be what we currently pass into the function; the latter would contain just the version specifier. So the updated signature could look similar to

`func newFakeMarathonEndpoint(t *testing.T, configs *ConfigContainer) *endpoint`

with `ConfigContainer` being something like

```
type ConfigContainer struct {
  client *config.Config
  server *config.ServerConfig
}
```

Since we currently only have a handful of cases where we actually pass a non-nil configuration to `newFakeMarathonEndpoint`, we wouldn't even need to change a lot of places to accommodate for the new prototype definition.

WDYT?